tests/data/test736

```diff
@@ -32,10 +32,10 @@ http
 </server>
 <setenv>
 HOME=%PWD/%LOGDIR
-IPFS_DATA=$HOME/.ipfs
+IPFS_PATH=$HOME/.ipfs
```

Comment:
To test that `IPFS_PATH` is really used by the curl tool, don't change `HOME`, and set `IPFS_PATH` to `%PWD/%LOGDIR/.ipfs`

(also in test737 and test738)

Reply:
Agreed, not setting HOME is preferred. Removed.